doc/sphinx/linux_install_guide/requirement.rst

```diff
@@ -91,8 +94,7 @@ to generate the configure script. Remember that we need Autoconf, Automake and L
 
     $ ./configure --enable-python
 
-The **--enable-python** option is absolutely necessary to install `Stimfit <http://www.stimfit.org>`_ since some of the functionality depends on Python. The configure script has some additional options. For example, we may want to use `IPython <http://www.scipy.org>`_  instead of the default embedded python shell with the option **---enable-ipython**  (note that the `IPython <http://www.scipy.org>`_ shell is only available under GNU/Linux and it is still very experimental). 
-
+The **--enable-python** option is absolutely necessary to install `Stimfit <http://www.stimfit.org>`_ since some of the functionality depends on Python. The configure script has some additional options. 
 
```

Comment:
Isn't `enable-python` activated by default? No need to explicitly add it in that case.

Reply:
I wasn't aware of that. Please, remove it.